app/(general)/integration/pooltogether-v4/layout.tsx

```diff
@@ -0,0 +1,61 @@
+'use client'
+import { ReactNode } from 'react'
+
+import classNames from 'clsx'
+import { motion } from 'framer-motion'
+import Image from 'next/image'
+import Link from 'next/link'
+import Balancer from 'react-wrap-balancer'
+
+import { FADE_DOWN_ANIMATION_VARIANTS } from '@/config/design'
+import { turboIntegrations } from '@/data/turbo-integrations'
+
+export default function GeneralLayout({ children }: { children: ReactNode }) {
+  const classes = classNames('GeneralLayout', 'bg-gradient-dark flex flex-col pb-10 lg:pb-12')
```

Comment:
what's the meaning of the `GeneralLayout` classname?

Reply:
Renamed to PoolTogetherLayout

Reply:
I meant this classname has no function, right? If so I think it should be removed. Ideally we just add tailwind utility classes to the classname